src/Model/GetInventoryRequestFromOrder.php

```diff
@@ -108,10 +108,10 @@ private function getAddressFromOrder(OrderInterface $order): ?AddressInterface
 
         return $this->addressInterfaceFactory->create([
             'country' => $shippingAddress->getCountryId(),
-            'postcode' => $shippingAddress->getPostcode() ?? '',
+            'postcode' => $shippingAddress->getPostcode() ? $shippingAddress->getPostcode() : '',
```

Comment:
![Screenshot 2023-12-08 at 16 04 19](https://github.com/AmpersandHQ/magento2-disable-stock-reservation/assets/600190/9af155ec-768e-47ac-9a33-fee1aded64d6)
https://www.php.net/manual/en/migration70.new-features.php

Not sure whats complaining here, this seems like an old way of doing it? or am i completely misinterpreting this.